src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -552,6 +552,11 @@ void HWIntrinsicInfo::lookupImmBounds(
                 immUpperBound = 7;
                 break;
 
+            case NI_Sve2_ShiftLeftAndInsert:
```

Comment:
you shouldn't need this because that should be already calculated as part of line 411 above under `HW_Category_ShiftLeftByImmediate`